tests/run/nogil.pyx

```diff
@@ -41,6 +41,12 @@ cpdef void release_gil_in_nogil2() nogil:
     with nogil:
         pass
 
+cdef int f_in_pxd1() nogil except -1:
+    return 0
+
+cdef int f_in_pxd2() nogil:  # implicit except -1?
+    return 0
+
```

Comment:
Can't these go to the end, so that we don't need to update all performance hint warning lines?